From ecd8e926bd364d866f492df87d7c46d029445480 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Tue, 7 Mar 2017 22:29:41 +0100
Subject: [PATCH] Add an option to disable NIS

NIS is not necessarily available in uClibc, so we need an option to
not compile support for it.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
[Peter: update for 2.7.16]
Signed-off-by: Peter Korsgaard <peter@korsgaard.com>
---
 configure.ac | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/configure.ac b/configure.ac
index 862c8ae516..354212fb16 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2857,6 +2857,12 @@ AC_ARG_ENABLE(codecs-cjk,
     	     DISABLED_EXTENSIONS="${DISABLED_EXTENSIONS} _codecs_kr _codecs_jp _codecs_cn _codecs_tw _codecs_hk"
   	  fi])
 
+AC_ARG_ENABLE(nis,
+	AS_HELP_STRING([--disable-nis], [disable NIS]),
+	[ if test "$enableval" = "no"; then
+    	     DISABLED_EXTENSIONS="${DISABLED_EXTENSIONS} nis"
+  	  fi])
+
 AC_SUBST(TK)
 AC_ARG_ENABLE(tk,
 	AS_HELP_STRING([--disable-tk], [disable tk]),
-- 
2.11.0

